packages/office-ui-fabric-react/src/components/DetailsList/ShimmeredDetailsList.styles.ts

```diff
@@ -1,13 +1,21 @@
+import { cssColor, IRGB } from '../../utilities/color/colors';
 import { IShimmeredDetailsListStyleProps, IShimmeredDetailsListStyles } from './ShimmeredDetailsList.types';
 
 export const getStyles = (props: IShimmeredDetailsListStyleProps): IShimmeredDetailsListStyles => {
-  const { theme, className, enableShimmer } = props;
-  const { palette } = theme;
+  const { theme, className, enableShimmer, showCheckbox, compact } = props;
+  const { semanticColors } = theme;
+
+  // Following lines are to convert the `listBackground` semantic color to a semitransparent color to fix a Safari bug
```

Comment:
Consider linking to a bug report as an audit trail for future maintainers if one is available.

Reply:
There is no issue officially tracking this and it's something that I noticed myself and also was told by one of the designers.

Reply:
I will file an issue not to forget to tackle this issue in a separate PR and revert the changes for now due to increased size in the bundle.